code/modules/atmospherics/machinery/pipes/pipes.dm

```diff
@@ -49,7 +49,7 @@
 		air_update_turf()
 
 /obj/machinery/atmospherics/pipe/return_air()
-	return parent.air
+	return parent ? parent.air : FALSE
```

Comment:
Just `parent?.air`, you shouldn't be returning FALSE here nothing expect boolean values here.